### PR TITLE
Install edxlocal debian packages

### PR DESCRIPTION
**Pull request by jzoldak**
@benpatterson @feanil

### CODE REVIEW

**Comment by benpatterson** [comment]
LGTM :+1:

File: playbooks/roles/edxlocal/tasks/main.yml [hunk 1]
```diff
@@ -2,6 +2,9 @@
 #
 ---
 
+- name: install packages needed for single server
+  apt: pkg={{','.join(edxlocal_debian_pkgs)}} install_recommends=yes state=present
```
Comment (e0d) [inline playbooks/roles/edxlocal/tasks/main.yml]: This idiom should go away in favor of with_items, which does the right thing.

Comment (benpatterson) [inline playbooks/roles/edxlocal/tasks/main.yml]: @e0d Updated in both places.

**Comment by feanil** [comment]
:+1:

**Pull request merged**